src/RpcServer/RpcServerPlugin.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+
+namespace Neo.Plugins
+{
+    public sealed class RpcServerPlugin : Plugin
+    {
+        static List<object> handlers = new List<object>();
+        RpcServer server;
+        RpcServerSettings settings;
+
+        protected override void Configure()
+        {
+            settings = new RpcServerSettings(GetConfiguration());
```

Comment:
We need to update the settings in `server` when this method was called. (Auto-reconfiguration)

Reply:
would it make sense to separate the settings that can be updated from those that cannot? Several of the settings (SslCert, SslCertPassword, MaxConcurrentConnections, TrustedAuthorities) are only used when creating the WebHost instance. If the user changes the RPC port after the system is running, the running WebHost port is not torn down and re-created.

Reply:
fixed